netutils/utils_linux.go

```diff
@@ -122,5 +122,6 @@ func FindAvailableNetwork(list []*net.IPNet) (*net.IPNet, error) {
 			}
 		}
 	}
-	return nil, fmt.Errorf("no available network")
+	// Failed to automatically select an available IP-range
+	return nil, fmt.Errorf("no available network, have tried: %+v", list)
```

Comment:
To be honest, Adding extra information of PredefinedLocalScopeDefaultNetworks which is "list" here doesn't provide any additional information to the user. 
1) predefined network is inbuilt or configured during init time.  We can't modify it. 
2) when we throw error saying "no available network", it means we are running out of  networks.
3) By providing inbuilt predefined networks list, user can't take any action other than deleting existing network and trying again.

Instead of  "have tried", you can mentioned "PredefinedLocalScopeDefaultNetworks List %+v"

Reply:
Thank you for your review and suggestions.
I have updated the code and squashed the commits.